### ISSUE TITLE
Android Wear integration

### ISSUE THREAD

**Issue by CaffeinatedAndroid** [google/google-authenticator-android#24]
Android Wear integration would make using 2-factor authentication a much simpler process when you don't have to hunt for your phone every time you want to log into a website/service. This feels like a significant omission in the Android ecosystem.

**Comment by ocdtrekkie** [comment]
Note that the published version of this app on the Play Store is closed source. Someone could submit a feature here, but I'm not sure that would translate to a feature there, as Google has moved away from the open source version.

**Comment by ThomasHabets** [comment]
Yeah. This is the opensource version. I wish github had template comments, since it's pretty much what I'd be adding to any feature request of this project;

This is the open source version of Google Authenticator, and is unlikely to ever get on the Play Store (though I think you are free to try to put it there, that's not my department). The Google Authenticator that's on the Play Store is closed source.

I would accept a pull request that cleanly implements this feature, but don't expect it from Google on this open source version. I can't comment on what will or won't happen to the Play Store version.

**Comment by kundan2607** [comment]
Hijddoh

**Comment by ibrahima** [comment]
FWIW, the closed source version has this now.
